docs/api/item.md

```diff
@@ -28,6 +32,125 @@ import Basic from '@site/static/usage/v7/item/basic/index.md';
 <Basic />
 
 
+## Content Types
+
+While items in a list take many forms, they typically support 5 different content types. However, not all of these content types should be used together at the same time. The following guide shows the different content types as well as how to properly utilize them in an application.
+
+### Supporting Visuals
+
+Supporting visuals are decorative icons or other adornments for an item. Common examples of supporting visuals are [Avatars](./avatar), [Icons](./icon), and [Thumbnails](./thumbnail). Since this content is not required to understand the intent of the item, it is typically hidden from screen readers using `aria-hidden="true"`.
+
+If a visual is required to interact with the item, such as an icon button, then the visual is an [action](#actions) not a supporting visual.
+
+<BestPracticeFigure 
+  text={<>Supporting visuals should be rendered in a consistent manner. This makes the information in each item easier to parse.</>}
+  doText={<>Render visuals in consistent slots</>}
+  dontText={<>Don't render visuals in different slots</>}
+  doImage={<img alt="A list with several items. Each item has an icon and visible text describing the item. The icon in each item is rendered at the start of the line." src={useBaseUrl('img/item/visuals-do.jpg')} />}
+  dontImage={<img alt="A list with several items. Each item has an icon and visible text describing the item. Some icons are rendered at the start of the line, and some icons are rendered at the end of the line" src={useBaseUrl('img/item/visuals-dont.jpg')} />}
+/>
+
+In the example below, we are creating two lists with supporting visuals. The first list uses icons, and the second list uses avatars. The visuals are decorative, so they all have `aria-hidden="true"`. Additionally, they are presented consistently in the `start` slot.
```

Comment:
It depends on the role of the visual. If the image is there as decoration, then it should have aria-hidden. However, if the image is required to understand the intent of the row then it should have alt text instead of aria-hidden.